src/components/process/manage/ManageProcess.js

```diff
@@ -42,6 +46,34 @@ export const ManageProcess = () => {
         })
     }
 
+    const startProcess = () => {
+        if (!process.endDate) {
+            Swal.fire({
+                title: "End date is not set!",
+                text: `Please, pick end date of recruitment process, as you won't be able to change it after process is started.`,
+                icon: 'error'
+            })
+        } else {
```

Comment:
No pytanie czy endDate nie chcielibyśmy móc zmienić już po otwarciu procesu bo jakby np nikt nie zaaplikował to racjonalne byłoby przedłużyć proces o kolejny tydzień/dwa niż otwierać nowy, ale to tak wg mnie XD